src/sentry/integrations/slack/message_builder/time_utils.py

```diff
@@ -0,0 +1,63 @@
+import time
+from collections.abc import Mapping
+from datetime import datetime, timedelta
+
+from django.utils import timezone
+from django.utils.timesince import timesince
+from django.utils.translation import gettext as _
+
+from sentry.models.group import Group
+
+DAY_IN_SEC = 24 * 60 * 60
+
+
+def get_approx_start_time(group: Group):
+    event = group.get_recommended_event_for_environments()
+    if event is None:
+        return None
+
+    occurrence = event.occurrence
+    if occurrence is None:
+        return None
+
+    regression_time = occurrence.evidence_data.get("breakpoint", None)
+    return regression_time
+
+
+def time_since(value: datetime):
+    """
+    Display the relative time
+    """
+    now = timezone.now()
+    if value < (now - timedelta(days=5)):
+        return value.date()
+    diff = timesince(value, now)
+    if diff == timesince(now, now):
+        return "Just now"
+    if diff == "1 day":
+        return _("Yesterday")
+    return f"{diff} ago"
+
+
+def get_relative_time(
```

Comment:
This is the only new method in this file. The logic is copied from frontend logic: https://github.com/getsentry/sentry/blob/master/static/app/components/events/eventStatisticalDetector/breakpointChart.tsx#L37